src/hotspot/share/gc/shared/stringdedup/stringDedupStat.cpp

```diff
@@ -94,13 +94,17 @@ static double strdedup_elapsed_param_ms(Tickspan t) {
 void StringDedup::Stat::log_summary(const Stat* last_stat, const Stat* total_stat) {
   log_info(stringdedup)(
     "Concurrent String Deduplication "
-    "%zu/" STRDEDUP_BYTES_FORMAT_NS " (new), "
+    "%zu (inspected), "
+    "%zu/" STRDEDUP_BYTES_FORMAT_NS " (new unknown), "
     "%zu/" STRDEDUP_BYTES_FORMAT_NS " (deduped), "
-    "avg " STRDEDUP_PERCENT_FORMAT_NS ", "
+    "total avg deduped/new unknown bytes " STRDEDUP_PERCENT_FORMAT_NS ", "
+    "%zu (total deduped), %zu (total new unknown), "
     STRDEDUP_ELAPSED_FORMAT_MS " of " STRDEDUP_ELAPSED_FORMAT_MS,
+    last_stat->_inspected,
     last_stat->_new, STRDEDUP_BYTES_PARAM(last_stat->_new_bytes),
     last_stat->_deduped, STRDEDUP_BYTES_PARAM(last_stat->_deduped_bytes),
     percent_of(total_stat->_deduped_bytes, total_stat->_new_bytes),
+    total_stat->_deduped_bytes, total_stat->_new_bytes,
```

Comment:
Shouldn't these two use `STRDEDUP_BYTES_FORMAT` and `STRDEDUP_BYTES_PARAM`?

Reply:
Yeah seems so,  this is what is used for `  STRDEDUP_BYTES_PARAM(last_stat->_new_bytes), STRDEDUP_BYTES_PARAM(last_stat->_deduped_bytes),`  .